js/isir-frame-io.js

```diff
@@ -6,26 +6,43 @@
 
 // Given a collection of [Files]() text files containing one or more fixed-width ISIR frames
 // Read each line, using CR/LF line endings, and return all non-blank, non SAIG routing codes.
-export async function read_isir_frames_from_filelist(file_list_src) {
+export async function read_isir_frames(filelist_src) {
     let isir_samples = []
-    for await (let isir_frame of aiter_read_isir_frames_from_filelist(file_list_src))
+    for await (let isir_frame of aiter_read_isir_frames(filelist_src))
         isir_samples.push(isir_frame)
     return isir_samples.flat()
 }
 
 
 // Given a collection of [Files]() text files containing one or more fixed-width ISIR frames
 // Read each line, using CR/LF line endings, and return all non-blank, non SAIG routing codes.
-export async function * aiter_read_isir_frames_from_filelist(filelist_src) {
-    const rx_saig_routing_header = /^[A-Z\*][0-9A-Z\*]+\s*,CLS=.*$/
-    const maybe_isir_frame = (ln) =>
-        ln.trim() && ! rx_saig_routing_header.test(ln)
+export async function * aiter_read_isir_frames(filelist_src) {
+    filelist_src = await filelist_src
```

Comment:
By assigning variables without const or let, the variable is defined in the global scope. `filelist_src` is a string and doesn't need to be awaiting, no?